backend/app/api/endpoints/admin.py

```
@@ -1707,3 +1715,643 @@ async def delete_public_retriever(
+        created_at=bot.created_at,
+        updated_at=bot.updated_at,
+        ghost_name=ghost_name,
+        shell_name=shell_name,
+        model_name=model_name,
+    )
+
+
+@router.delete("/public-bots/{bot_id}", status_code=status.HTTP_204_NO_CONTENT)
+async def delete_public_bot(
+    bot_id: int = Path(..., description="Bot ID"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)